src/components/Modals/ConfirmModal/ConfirmModal.tsx

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable no-restricted-imports */
+/* External dependencies */
+import React, {
+  useCallback,
+  forwardRef,
+} from 'react'
+
+/* Internal dependencies */
+import { Modal } from '../Modal'
+import { ConfirmModalProps } from './ConfirmModal.types'
+
+function ConfirmModal(
+  {
+    children,
+    onHide,
+    onConfirm,
+    onKeyDown,
+    ...rests
+  }: ConfirmModalProps,
+  forwardedRef: React.Ref<HTMLDivElement>,
+) {
+  const handleOnKeyDown = useCallback((e: React.KeyboardEvent<HTMLDivElement>) => {
```

Comment:
```suggestion
  const handleKeyDown = useCallback((e: React.KeyboardEvent<HTMLDivElement>) => {
```
보통 on{event} 에 대응하는 함수를 작성할 때 handle{event}로 네이밍 합니다.